src/constants/fieldTypesNew.js

```diff
@@ -100,14 +100,14 @@ export const fieldTypeInfo = {
     initialDefaultValue: null,
     exampleValue: 12,
   },
-  [fieldTypes.file]: {
-    labelId: 'FILE_UPLOADER',
-    value: fieldTypes.file,
-    backendType: backendTypes.string,
-    backendMultiple: false,
-    canProvideDefaultValue: false,
-    exampleValue: null,
-  },
+  // [fieldTypes.file]: {
+  //   labelId: 'FILE_UPLOADER',
+  //   value: fieldTypes.file,
+  //   backendType: backendTypes.string,
+  //   backendMultiple: false,
+  //   canProvideDefaultValue: false,
+  //   exampleValue: null,
+  // },
```

Comment:
this is broken and without backend support. commenting out for now since tanya says we will be working on this feature soon